ui/shared/StyledText.qml

```diff
@@ -2,5 +2,5 @@ import QtQuick 2.13
 import "../imports"
 
 Text {
-    font.family: Theme.fontRegular.name
+    font.family: Theme.fontEmoji.name + "," + Theme.fontRegular.name
```

Comment:
How does it work? Doesn't it work like HTML where it uses the first font, if it doesn't find it, it uses the second one?